SPECS/openvswitch/openvswitch.spec

```diff
@@ -200,7 +200,18 @@ make -k check |& tee %{_specdir}/%{name}-check-log || %{nocheck}
 %{_unitdir}/openvswitch.service
 %{_unitdir}/ovs-vswitchd.service
 %{_unitdir}/ovsdb-server.service
-%{_libdir}/lib*
+%{_libdir}/libovn.so
+%{_libdir}/libovn-2.12.so*
```

Comment:
Awesome, thanks for breaking it down this way! Please include the first number after "so.", though. For example replace `%{_libdir}/libovn-2.12.so*` with `%{_libdir}/libovn-2.12.so.0*`.

Why: this helps us catch any shared library version changes, which would then require a release number bump to packages linking against these libraries, so that we build a newer version linked against the updated libraries. It's a nice build-time check [Fedora is doing](https://docs.fedoraproject.org/en-US/packaging-guidelines/#_shared_libraries) for instance.